internal/common/gamestate/gamestate_test.go

```diff
@@ -39,8 +39,8 @@ func TestGetClientGameStateCopy(t *testing.T) {
 				CriticalConsecutiveTurnsCounter: 2,
 			},
 		},
-
-		CommonPool: 20,
+		IIGORolesBudget: map[shared.Role]shared.Resources{},
```

Comment:
You probably want to put some values in this.

Reply:
whoops! i'll make another PR! sorry about this

Reply:
np. its just the tests you put in arent testing properly, but im sure functionality wise it works :)

Reply:
but please do make another pr to fix this :D